cgroup.go

```diff
@@ -60,6 +77,16 @@ func Load(hierarchy Hierarchy, path Path) (Cgroup, error) {
 			if os.IsNotExist(errors.Cause(err)) {
 				return nil, ErrCgroupDeleted
 			}
+			if err == ErrControllerNotActive {
```

Comment:
Do we need to change the location for `if os.IsNotExist(errors.Cause(err))` and  `if err == ErrControllerNotActive`, like
```
			if err == ErrControllerNotActive {
				for _, o := range skip {
					if skerr := o(s, path, err); skerr != nil {
						if skerr != ErrSkipSubsystem {
							return nil, skerr
						}
					}
				}
				continue
			}
                        if os.IsNotExist(errors.Cause(err)) {
				return nil, ErrCgroupDeleted
			}
```
first check controller is support, then check the file is exist ?

Reply:
I think this is fine to keep the existing logics for Load, the isnotexist is for when the cgroup directory structure has been removed, not an active controller.  I think this is fine for the Load case to get the ErrCgroupDeleted error which is handled